packaging/ubuntu-16.04/snapd.manpages

```diff
@@ -1 +1,2 @@
+snapctl.1
```

Comment:
Nothing for Ubuntu 14.04?

Reply:
Good catch, missed the fact we have separate builds. Fixed.